opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -597,6 +596,7 @@ class Well {
     PAvg m_pavg;
     double well_temperature;
     InjMultMode inj_mult_mode = InjMultMode::NONE;
+    InjMult well_inj_mult;
```

Comment:
Is it reasonable to say that all wells always have an `InjMult` state?  Could you perhaps use
```C++
std::optional<InjMult> well_inj_mult
```
instead?  Then `nullopt` would/could represent "there's no `WINJMULT` setting pertaining to this well" and you might not need the `NONE` mode at all.

Reply:
It is possible, let me try to update it. It can also apply possibly to the Connection side also.

Reply:
I changed the `InjMult` members to be `std::optinal<InjMult>` for both `Well` and `Connection`, while keeping the `NONE` in `InjMultMode`, mainly for being able to have a value to be able to initialize `InjMultMode`. 

Otherwise, we can also make `InjMultMode` to be `std::optional` to remove `NONE` type in `InjMultMode`. Let me know if you have some comments along this line.